tests/widget/inline/widget-inline.webauthn-login.test.js

```diff
@@ -0,0 +1,38 @@
+import { expect, test } from '@playwright/test';
+import { asyncEvents, verifyUserInfo } from '../../utilities/async-events.js';
+
+test('inline widget with webauthn login', async ({ browser, page }) => {
+  const { clickButton, navigate } = asyncEvents(page);
+
+  let authenticator;
+  if (browser.browserType().name() === 'chromium') {
+    const cdpSession = await page.context().newCDPSession(page);
+
+    await cdpSession.send('WebAuthn.enable');
+    authenticator = await cdpSession.send('WebAuthn.addVirtualAuthenticator', {
+      options: {
+        protocol: 'ctap2',
+        transport: 'internal',
+        hasUserVerification: true,
+        isUserVerified: true,
+        hasResidentKey: true,
+      },
+    });
+
+    await navigate('widget/inline?journey=TEST_WebAuthn-Registration');
+
+    await page.getByLabel('Username').click();
+    await page.getByLabel('Username').fill('demouser');
+    await page.getByLabel('Username').press('Tab');
+
+    await page.getByLabel('Password').fill('j56eKtae*1');
+
+    await clickButton('Next', '/authenticate');
+
+    await verifyUserInfo(page, expect);
+
+    await cdpSession.send('WebAuthn.removeVirtualAuthenticator', {
+      authenticatorId: authenticator.authenticatorId,
+    });
```

Comment:
Where are we actually testing the success of the WebAuthn?

Reply:
Followed the flow of others test cases by using the **verifyUserInfo(page, expect)** function

Reply:
I'm guessing that the CDP is able to detect the prompt for a security key/finger print and then it automatically succeeds?